cookbook/testing.md

```diff
@@ -96,27 +93,27 @@ const wrapper = mount(Counter, {
   },
 })
 
-const store = useSomeStore() // uses the testing pinia!
+const store = useSomeStore() // ¡utiliza la pinia de pruebas!
 
-// state can be directly manipulated
+// el estado puede manipularse directamente
 store.name = 'my new name'
-// can also be done through patch
+// también se puede hacer a través de patch
 store.$patch({ name: 'new name' })
 expect(store.name).toBe('new name')
 
-// actions are stubbed by default, meaning they don't execute their code by default.
-// See below to customize this behavior.
+// están bloqueadas por defecto, lo que significa que no ejecutan su código por defecto.
```

Comment:
"**las acciones** están bloqueadas por defecto" instead of "están bloqueadas por defecto"

Reply:
done